webpack.config.js

```diff
@@ -42,7 +42,7 @@ module.exports = (env = {}) => {
       new CleanWebpackPlugin(),
       new DefinePlugin({
         VERSION: JSON.stringify(packageJson.version),
-        DEVELOP: env.development,
+        DEVELOP: env.development ? true : false,
```

Comment:
I'd prefer it like this:

```suggestion
        DEVELOP: !!env.development,
```

Reply:
Yes, that's possible. I've just created another commit and performed a push operation.